example/full-config.php

```diff
@@ -193,6 +137,16 @@
 
             DependencyFactory::class => DependencyFactoryFactory::class,
             ConfigurationLoader::class => ConfigurationLoaderFactory::class,
+
+            FilesystemAdapter::class => static function (ContainerInterface $container): FilesystemAdapter {
+                $config = $container->get('config');
+                $params = $config['doctrine']['cache']['filesystem'];
+
+                return new FilesystemAdapter(
+                    namespace: $params['namespace'],
+                    directory: $params['directory'],
```

Comment:
Would be best to avoid named arguments here, for compatibility with any potential upstream change.

The usage without named arguments seems very clear.